client/components/AdoptionForm.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from "react"
+const homeStatus = ["", "Home Owner", "Renter"]
+
+const AdoptionForm = props => {
+    const [formData, setFormData] = useState({
+        name: "",
+        phoneNumber: "",
+        email: "",
+        homeStatus: ""
+    })
+    const homeStatusOptions = homeStatus.map(option => {
+        return (
+            <option key={option} value={option}>{option}</option>
+        )
+    })
+    const onSubmitHandler = event => {
+        event.preventDefault()
+        props.onAdoptionFormSubmitted(formData)
+    }
+    const handleInputChange = event => {
+        setFormData({
+          ...formData,
+          [event.currentTarget.id]: event.currentTarget.value
+        })
+    }
+    return (
+        <form onSubmit={onSubmitHandler}>
```

Comment:
We can add empty lines for readability between where one function ends and another begins.
Let's add lines between 10&11, 15&16, 19&20, 25&26